### PR TITLE
Test timeselection

### PR DESCRIPTION
**Pull request by jorana**
**What is the problem / what does the code in this PR do**
See https://github.com/AxFoundation/strax/pull/345. Here Daniel suggested (https://github.com/AxFoundation/strax/pull/345#pullrequestreview-541531692) to test. This is not that easy to do in strax but very doable in straxen. It also kind of makes sense to do it here because it strongly depends on the way one sets up the `plugins`, their `dependencies` and `plugin.save_when`.

**Can you briefly describe how it works?**
I manually extracted 13 offending time ranges. Try running into an error by forcing these time ranges. 

**Can you give a minimal working example (or illustrate with a figure)?**
If all goes well, this PR will fail first try (until we merge 345).

### CODE REVIEW

**Comment by jorana** [comment]
Great, 0ddecd3 failed, 4d2f818 was committed **after** merging AxFoundation/strax#345 and should therefore succeed, highlighting the issue that this PR is addressing.

File: tests/test_strax_time_selection.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"""
```
Comment (WenzDaniel) [inline tests/test_strax_time_selection.py:1]: Maybe add this text to your assert message.

**Review by WenzDaniel — APPROVED** [review]
Looks nice, thank you very much.

File: tests/test_strax_time_selection.py (hunk 1 above)
Comment (jorana) [reply tests/test_strax_time_selection.py:1]: Thanks, the assert statement should simply never fail it's not really the test itself. Nonetheless added a sting just above the - would be failing - line

**Comment by jorana** [comment]
Added another 6% coverage :)

**Pull request merged**